autodistill/helpers.py

```diff
@@ -1,14 +1,58 @@
 import os
 import random
 import shutil
+from io import BytesIO
+from typing import Any
 
 import cv2
+import numpy as np
+import requests
 import supervision as sv
 import tqdm
 import yaml
 from PIL import Image
 
 VALID_ANNOTATION_TYPES = ["box", "mask"]
+ACCEPTED_RETURN_FORMATS = ["PIL", "cv2", "numpy"]
+
+
+def load_image(
+    image: Any,
+    return_format="cv2",
+) -> Any:
+    if return_format not in ACCEPTED_RETURN_FORMATS:
+        raise ValueError(f"return_format must be one of {ACCEPTED_RETURN_FORMATS}")
+
+    if isinstance(image, Image.Image) and return_format == "PIL":
+        return image
+    elif isinstance(image, Image.Image) and return_format == "cv2":
+        return np.array(image)
```

Comment:
I think with PIL to cv2 you have to convert the channels from RGB to BGR? (And vice versa for cv2 to others)

Reply:
I have added the requisite conversion code in my latest commit.